polylogue/util.py

```
@@ -161,6 +175,30 @@ def _set_meta_value(key: str, value: str) -> None:
+    digest = hashlib.sha256(material).hexdigest()
+    return f"import.sync.{digest}"
+
+
+def get_import_sync_state(provider: str, conversation_id: str) -> Optional[dict]:
+    raw = _get_meta_value(import_sync_state_key(provider, conversation_id))
+    if not raw:
+        return None
+    try:
+        data = json.loads(raw)
+        return data if isinstance(data, dict) else None
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)